router/middleware.go

```diff
@@ -53,6 +54,22 @@ func getSessionFromCookieOrHeader(c *gin.Context) {
 	c.Next()
 }
 
+func _debugBypassEmptyLogin(c *gin.Context) {
+	if c.GetBool("loggedIn") {
+		c.Next()
+		return
+	}
+
+	bypassLogin := os.Getenv("RFD_BYPASS_EMPTY_LOGIN") == "true" && gin.IsDebugging()
```

Comment:
So for this…

Having to evaluate the env variable each time is a bit weird to me.

Here I’d pass this value in to the function.  Then return middleware function.


A pattern like:
```
func myMiddleware(enable bool) func (*gin.Context) {
    // do some logic or set to a variable
    return func(c *gin.Context) {
      // code actually executed as middleware
    }
} 
```

Then can use by:
```
router.Use(myMiddleware(passBoolOrVariable)
```

Reply:
Also purely nitpick.  Can we do: RFD_DISABLE_LOGIN_VERIFY maybe?